onnx_tf/handlers/backend/conv_integer.py

```diff
@@ -0,0 +1,60 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from .conv_mixin import ConvMixin
+
+
+@onnx_op("ConvInteger")
+class Conv(ConvMixin, BackendHandler):
+
+  @classmethod
+  def _apply_x_zero_point(cls, base, zero_point):
+    base = tf.cast(base, tf.float32)
+    zero_point = tf.cast(zero_point, tf.float32)
+    return (base - zero_point)
+
+  @classmethod
+  def _apply_w_zero_point(cls, base, zero_point):
+    if len(zero_point.shape) == 0:
+      w_zero_point = cls._apply_x_zero_point(base, zero_point)
+    elif len(zero_point.shape) == 1:
+      for i in range(base.shape.as_list()[0]):
+        for j in range(zero_point.shape.as_list()[0]):
+          tensor = cls._apply_x_zero_point(base[i][j], zero_point[j])
+          if "out_tensor" in locals():
+            out_tensor = tf.concat([out_tensor, tensor], 0)
+          else:
+            out_tensor = tensor
+      w_zero_point = tf.reshape(out_tensor, base.shape)
+    else:
+      raise ValueError("Unsupported zero point: {}".format(zero_point))
+
+    return w_zero_point
+
+  @classmethod
+  def version_10(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    x = tensor_dict[node.inputs[0]]
+    w = tensor_dict[node.inputs[1]]
+
+    # Apply zero_point to x and w
+    x = cls._apply_x_zero_point(x, tensor_dict[node.inputs[node.inputs.index(
```

Comment:
consider to simplify 'tensor_dict[node.inputs[node.inputs.index("x_zero_point")]]' to 'tensor_dict["x_zero_point"]' for line 42 and also line 46